src/main/java/uk/gov/hmcts/cmc/claimstore/events/claimantresponse/ClaimantResponseActionsHandler.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.cmc.claimstore.events.claimantresponse;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.ClaimantResponse.referenceForDefendant;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.DEFENDANT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.FRONTEND_BASE_URL;
+
+@Component
+public class ClaimantResponseActionsHandler {
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    public ClaimantResponseActionsHandler(
+        NotificationService notificationService,
+        NotificationsProperties notificationsProperties
+    ) {
+        this.notificationService = notificationService;
+        this.notificationsProperties = notificationsProperties;
+    }
+
+    @EventListener
+    public void sendNotificationToDefendant(ClaimantResponseEvent event) {
+        Claim claim = event.getClaim();
+        this.notificationService.sendMail(
```

Comment:
I think we can reuse `NotificationToDefendantService` to send this email, as code below is exactly sending the same email template.

Reply:
I agree with you Sabah, however if you have to change the template, then it need to reworked as now.  if Tim is changing for this, then he need to refactor all places even in ccd.

Reply:
done, didn't see that before

Reply:
Thanks @timja 
 @dharmendrak  `CCJ` cleanup can be done in separate PR IMO